### DISCUSSION TITLE
Windows: open file on another drive

### DISCUSSION THREAD

**Discussion by sybrenstuvel** [vlachoudis/bCNC#1821] [Q&A]
Hey folks,

I'm trying out bCNC for the first time, on Windows 10. When I click the "Open" icon to open a GCode file, it starts the file dialog at the current working directory, `C:\workspace\bCNC`. How do I change to another drive?

![image](https://user-images.githubusercontent.com/1092128/224482254-f74bf792-830b-442d-a396-d9471941e67b.png)

I can click on the path components, and go all the way up to the `C:\` root, but I can't figure out how to change to `D:\`, where my GCode lives. Contrary to most other file dialog boxes, I also cannot type a filename or directory path to go to, so I'm forced to click my way there.

Cheers,
Sybren

**Comment by JipeB** [comment]
Hi,
Try typing D:/ in the filename field

JP

**Reply by sybrenstuvel** [reply]
This is why I wrote:
> Contrary to most other file dialog boxes, I also cannot type a filename or directory path to go to, so I'm forced to click my way there.
